src/game_updater/GameUpdater.cpp

```diff
@@ -167,28 +168,58 @@ void GameUpdater::UpdatePlayer(std::shared_ptr<GameState> game_state) {
   }
 }
 
+std::shared_ptr<Platform> GetCurrentPlatform(std::shared_ptr<Level> level) {
+  return level->getPlatforms()->at(std::max(
+      0l,
+      std::min((long)(level->getMusic()->getPlayingOffset().asMilliseconds() /
+                      (float)MS_PER_PLATFORM),
+               (long)level->getPlatforms()->size() - 1)));
+}
```

Comment:
@bbeck13 Would it be possible to rewrite this later to use the elapsed ticks instead of directly accessing the music?